Source/Lib/Common/Codec/EbThreads.c

```diff
@@ -31,12 +31,16 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <pthread.h>
-#include <semaphore.h>
 #include <unistd.h>
-#endif // _WIN32
 #ifdef __APPLE__
 #include <dispatch/dispatch.h>
-#endif
+typedef dispatch_semaphore_t sem_t;
+INLINE int sem_wait(sem_t* psem) { return (int)dispatch_semaphore_wait(*psem, DISPATCH_TIME_FOREVER); }
```

Comment:
```suggestion
static inline int sem_wait(sem_t* psem) { return (int)dispatch_semaphore_wait(*psem, DISPATCH_TIME_FOREVER); }
```

static inline might be better to reduce the amount of generic global function names we might have.

Reply:
@1480c1 , thanks for suggestion. 
I will change it later.